### ISSUE TITLE
Unitless quantities should convert to primitives

### ISSUE THREAD

**Issue by nickbabcock** [iliekturtles/uom#64]
I expected something along the lines of the following to work:

```rust
let start = Time::new::<second>(1.0);
let end = Time::new::<second>(10.0);
let interval = Time::new::<second>(3.0);

let buckets: f32 = ((end - start) / interval).into()
```

I'm curious if I'm missing something, as I'm still exploring the library. Currently the workaround seems to be

```rust
(end - start).get(second) / interval.get(second)
```

Which isn't as pretty 😄

**Comment by iliekturtles** [comment]
The way to do this right now is to directly access the `value` member of the `Quantity` struct. However I would like to make `value` private in the future so a better way is needed.

I haven't put much thought into dimensionless quantities at this point as I wanted to complete #3. This issue touches on dimensionless quantities because similar to the difference between `ThermodynamicTemperature` and `TemperatureInterval` there are multiple distinct dimensionless types: `Ratio`, as seen here, but also types like `Angle` (m/m) and `PowerEfficiency` (W/W).

I'll leave the issue open with the intention of implementing `impl<U, V> From<V> for Quantity<DimensionOne, U, V>`.

```rust
    let start = Time::new::<second>(1.0);
    let end = Time::new::<second>(10.0);
    let interval = Time::new::<second>(3.0);

    let buckets: f32 = ((end - start) / interval).value;

    println!("{}", buckets);
```

**Comment by nickbabcock** [comment]
The temporary workaround worked great. uom helped me solve a couple head scratching bugs.

I'm excited for when dimensionless quantities become first class citizens 😄

**Comment by iliekturtles** [comment]
Awesome! Glad to hear it helped solve some problems for you!

**Comment by shradej1** [comment]
Is there a dimensionless unit that can be used to get at the value in a unit-safe way?  Like,

    let buckets: f32 = ((end - start) / interval).get(unit);

**Comment by iliekturtles** [comment]
Not yet, but there will be "soon." I have most of the changes complete in the [`dev-ratio`](https://github.com/iliekturtles/uom/tree/dev-ratio) branch to create a new `Ratio` quantity. You'll be able to do the following once it is merged:

```rust
let buckets: f32 = ((end - start) / interval).get(ratio);
```

**Comment by shradej1** [comment]
That's great!  An `Into` conversion to primitives might be nice too, but I think the ratio approach is a more consistent treatment.

**Comment by iliekturtles** [comment]
`From`/`Into` will also be available for `Ratio`s once changes are merged into master! https://github.com/iliekturtles/uom/blob/7dec1f186d3080b3726a4964176c53f9af8fc3ae/src/si/ratio.rs#L35
